tests/determine-basal.test.js

```diff
@@ -484,10 +487,10 @@ describe('determine-basal', function ( ) {
       result.error.should.equal('Error: iob_data undefined. ');
     });
 
-    it('iob_data should contain activity, iob, bolussnooze', function () {
-      var result = determine_basal({glucose:100}, undefined,{"activity":0}, {"current_basal":0.0, "max_bg":100,"min_bg":110}, undefined, meal_data, tempBasalFunctions);
-      result.error.should.equal('Error: iob_data missing some property. ');
-    });
+    //it('iob_data should contain activity, iob, bolussnooze', function () {
```

Comment:
unit tests should be maintained with the code. not commenting out unit test if new test break the unit tests

Reply:
Bolus snooze was removed, so I removed the tests that validate it as well.  If you think this test should be replaced with another one, please feel free to PR it in.